### DISCUSSION TITLE
<Intermediate Git>: <Implement Commit Abbreviations suggestion into a >

### DISCUSSION THREAD

**Discussion by CBID2** [TheOdinProject/curriculum#25710] [Ideas]
### Describe your suggestion

Hi @KevinMulhern. Remember when you made [this suggestion when I worked on the Commits lesson](https://github.com/TheOdinProject/curriculum/pull/25673/files/92f95f37f4a31e332f95743decf7880c40f134d6#r1244396541)? Well, I think would be beneficial to add them somewhere in the Intermediate Git section as a separate lesson or into the "A Deeper Look at Git" lesson(personally, I think it should go here because it seems to use a scaffolding approach before it goes into detail about the other aspects of Git).   

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-a-deeper-look-at-git

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Chrissy

**Comment by KevinMulhern** [comment]
Thanks for following up with this @CBID2. I’m not 100% sure it fits with that lesson, it goes deeper into Git commands whereas commit types seems more like a style thing.

Thinking about it some more - types of commits are typically directly related to the story/task the commit is associated with on a project management tool. 

For example, on the Odin Project web app, we have three types of stories we use on [our project board](https://github.com/orgs/TheOdinProject/projects/12) and those dictate what we use for the commit type:

* Feature/Enhancement - Things that change the product in some way. This could be new features, UI improvements, performance improvements etc. Basically anything a user can see and interact with.
* Chore - Things that have to be done, but don’t change the product. Upgrading dependencies, refactoring etc
* Fix - Anything that is broken / not behaving as intended, bug fixes.

All that is to say, I don't think learners will get much value out of been taught about commit types without having a basic understanding of project management in software development first.

If we had an agile / project management lesson, this would fit in perfectly on it. But we don’t have anything like that yet unfortunately.

**Comment by CBID2** [comment]
Ahh ok thanks @KevinMulhern. Let me know when you create that lesson

**Comment by KevinMulhern** [comment]
Will do @CBID2, it's not something we're prioritising right now, but offering lessons on basic project management is something we'd like to do someday. 

Moved this to a discussion because it's not something we can work on straight away.

**Reply by CBID2** [reply]
Got it
